### PR TITLE
Update Data Factories Overview for .NET

### PR DESCRIPTION
**Pull request by JeremyLikness**
Add overview text with links to related services and introduction, add links to tutorials, document management library with example, show link to sample code and add boilerplate samples link. Tagging @CamSoper for review and @JennieHubbard as listed manager. CDA team will review for sign-off tag.

### CODE REVIEW

**Comment by openpublishbuild** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/data-factory.md](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-3/api/overview/azure/data-factory.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/dotnet/api/overview/azure/data-factory?branch=pr-en-us-247) | [Details](#user-content-api%2Foverview%2Fazure%2Fdata-factory.md)

<a name="api%2Foverview%2Fazure%2Fdata-factory.md"></a>
### [api/overview/azure/data-factory.md](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-3/api/overview/azure/data-factory.md)
 - **Line 24**: **[Warning]** Invalid file link:(~/api/overview/azure/data-factory/data-factory-introduction).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C20%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201707201900435323-247%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-3/api/overview/azure/index.yml) | :x:Error | | [Details](#user-content-api%2Foverview%2Fazure%2Findex.yml)
[]() | :x:Error | | [Details](#user-content-)

<a name="api%2Foverview%2Fazure%2Findex.yml"></a>
### [api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-3/api/overview/azure/index.yml)
 - **[Error]** Error transforming model &quot;W:\3k4w-o-a4d6583d\raw-\zj1c1ro4.usb\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

<a name=""></a>
### []()
 - **[Error]** Error transforming model &quot;W:\3k4w-o-a4d6583d\raw-\zj1c1ro4.usb\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C20%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201707202247552484-247%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api/overview/azure/data-factory.md [hunk 1]
```diff
@@ -38,19 +45,31 @@ dotnet add package Microsoft.Azure.Management.DataFactories
...
 ```
 
 > [!div class="nextstepaction"]
 > [Explore the management APIs](/dotnet/api/overview/azure/datafactories/management)
 
-
 ## Samples
 
---Samples list here--
+* [MyDriving - An Azure IOT and Mobile Sample Application](https://azure.microsoft.com/en-us/resources/samples/mydriving/) that uses Data Factory to drive insights.
```
Comment (MaximRouiller) [inline api/overview/azure/data-factory.md]: en-us link should be removed.

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-3/api/overview/azure/index.yml) | :x:Error | | [Details](#user-content-api%2Foverview%2Fazure%2Findex.yml)
[]() | :x:Error | | [Details](#user-content-)

<a name="api%2Foverview%2Fazure%2Findex.yml"></a>
### [api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-3/api/overview/azure/index.yml)
 - **[Error]** Error transforming model &quot;W:\mjkc-o-2e097070\raw-\djoop5xo.x0i\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

<a name=""></a>
### []()
 - **[Error]** Error transforming model &quot;W:\mjkc-o-2e097070\raw-\djoop5xo.x0i\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C21%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201707211241572240-247%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by MaximRouiller** [comment]
#Sign-off

**Pull request merged**